packages/compass/src/app/index.tsx

```diff
@@ -30,14 +30,17 @@ window.addEventListener('error', (event) => {
   );
 });
 
-window.addEventListener('unhandledrejection', (event) => {
-  event.preventDefault();
-  const error = ensureError(event.reason);
-  void ipcRenderer?.call('compass:error:fatal', {
-    message: error.message,
-    stack: error.stack,
-  });
-});
+window.addEventListener(
+  'unhandledrejection',
+  (event: PromiseRejectionEvent) => {
+    event.preventDefault();
```

Comment:
Why `preventDefault()`?

Reply:
I copy/pasted it from the error handler. The only difference is in getting an error object from a promise rejection.